pkg/arvo/app/chat-pull-hook.hoon

```diff
@@ -0,0 +1,260 @@
+/-  *invite-store, hook=chat-pull-hook
+/+  default-agent, verb, dbug, store=chat-store, lib=chat-hooks
+|%
++$  card  card:agent:gall
++$  versioned-state
+  $%  [%0 state-0]
+  ==
++$  state-0
+  $:  tracking=(map path ship)
+  ==
+--
+::
+=|  [%0 state-0]
+=*  state  -
+::
+%-  agent:dbug
+%+  verb  |
+::
+^-  agent:gall
+|_  bol=bowl:gall
```

Comment:
Please remove bol and just call this bowl. Removes possibility of error/confusion in calling `bowl` directly.

Reply:
Isn't the whole problem with autonaming error/confusion with calling `bowl` directly?

Reply:
@ohAitch I think @loganallenc means `=bowl:gall`, equivalent to `bowl=bowl:gall`? We use this all over the place these days, and it's not a problem since you generally don't refer to the bowl type elsewhere, and it'd still be at `bowl:gall` if you needed it.